blog/2025-06-openapi-generator/README.md

```diff
@@ -0,0 +1,465 @@
+| `npmName`                        | Required: name of the npm package. |
+| `npmRepository`                  | URL for a private npm repository (for `publishConfig`). |
+| `npmVersion`                     | Version of the npm package. Default: `1.0.0`. |
+| `nullSafeAdditionalProps`       | Indexers for `additionalProperties` return `undefined`. Default: `false`. |
+| `paramNaming`                    | Parameter naming: `camelCase`, `PascalCase`, `snake_case`, `original`. Default: `camelCase`. |
+| `prependFormOrBodyParameters`    | Put body/form params first. Default: `false`. |
+| `providedIn`                     | Injection scope: `root`, `none`, `any`, `platform`. Default: `root`. |
+| `queryParamObjectFormat`         | Format for query object: `dot`, `json`, `key`. Default: `dot`. |
+| `rxjsVersion`                    | Compatible RxJS version. |
+| `serviceFileSuffix`              | File suffix for service files. Default: `.service`. |
+| `serviceSuffix`                  | Suffix for service class names. Default: `Service`. |
+| `snapshot`                       | Adds timestamp to version (e.g. `1.0.0-SNAPSHOT.20250615`). Default: `false`. |
+| `sortModelPropertiesByRequiredFlag` | Required model props come first. Default: `true`. |
+| `sortParamsByRequiredFlag`       | Required method args come first. Default: `true`. |
+| `stringEnums`                    | Use TypeScript string enums. Default: `false`. |
+| `supportsES6`                    | Use ES6-compatible syntax. Default: `false`. |
+| `taggedUnions`                   | Use tagged unions via discriminators. Default: `false`. |
+| `tsVersion`                      | Compatible TypeScript version. |
+| `useSingleRequestParameter`      | Group all method parameters into a single object. Default: `false`. |
+| `useSquareBracketsInArrayNames`  | Append `[]` to array param names. Default: `false`. |
+| `withInterfaces`                 | Generate interfaces alongside classes. Default: `false`. |
+| `zonejsVersion`                  | Compatible version of `zone.js`. |
+-->
+
+In the next section, we'll explore how to build and integrate this code into your Angular project.
+
+
+## Set Up the Generated Code
+
+To integrate the generated API client into your Angular application, simply register the client using `provideApi()` inside your `app.config.ts`. 
+This setup is fully compatible with Angular's standalone application structure:
+
+```ts
+// app.config.ts
+import { ApplicationConfig } from '@angular/core';
+import { provideHttpClient } from '@angular/common/http';
+import { provideApi } from './shared/book-monkey-api';
+
+export const appConfig: ApplicationConfig = {
+  providers: [
+    // [...]
+    provideHttpClient(),
+    provideApi()
+  ]
+};
+```
+
+Don't forget to include `provideHttpClient()`.
+It registers Angular's HttpClient for injection.
+This is required, because the generated client uses HttpClient internally for every request.

```

Comment:
```suggestion
Don't forget to include `provideHttpClient()` to register Angular's `HttpClient` for injection.
This is required, because the generated client uses `HttpClient` internally for every request.
```